packages/mantine/src/suggestionMenu/SuggestionMenuItem.tsx

```diff
@@ -1,65 +1,45 @@
 import * as Mantine from "@mantine/core";
 
+import { assertEmpty } from "@blocknote/core";
 import { ComponentProps } from "@blocknote/react";
-import { forwardRef, useCallback } from "react";
+import { forwardRef } from "react";
 
 export const SuggestionMenuItem = forwardRef<
   HTMLDivElement,
   ComponentProps["SuggestionMenu"]["Item"]
 >((props, ref) => {
-  const {
-    className,
-    title,
-    subtext,
-    // group,
-    icon,
-    badge,
-    // aliases,
-    // onItemClick,
-    isSelected,
-    setSelected,
-    onClick,
-  } = props;
+  const { className, isSelected, onClick, item, id, ...rest } = props;
 
-  const handleMouseLeave = useCallback(() => {
```

Comment:
How come we don't need this anymore?

Reply:
I've disabled this because I figured copying the ariakit combobox behavior (separate focus and hover states) would be fine.

But then I noticed for menus they don't have this behavior, so I started a conversation with the ariakit maintainer:

<img width="606" alt="image" src="https://github.com/TypeCellOS/BlockNote/assets/368857/35ffad38-dfe8-44f1-9342-71f04c870d45">

Perhaps bringing it back is a good idea, but let's do this as a separate PR or if more people experience this as an issue, as we're a bit limited on time atm